lib/Doctrine/DBAL/Driver/DrizzlePDOMySql/Connection.php

```diff
@@ -10,12 +10,12 @@ class Connection extends PDOConnection
     /**
      * {@inheritdoc}
      */
-    public function quote($value, $type = ParameterType::STRING)
```

Comment:
Let's use `$value` everywhere. The fact that only input needs to be quoted is $1M misconception.